cfy_cluster_setup/main.py

```diff
@@ -37,8 +37,10 @@
 
 CA_PATH = join(CERTS_DIR, 'ca.pem')
 
-CLUSTER_CONFIG_FILE_NAME = 'cfy_cluster_config.yaml'
 CREDENTIALS_FILE_PATH = join(os.getcwd(), 'secret_credentials.yaml')
+CLUSTER_CONFIG_FILES_DIR = join(dirname(__file__), 'cfy_cluster_config_files')
+THREE_NODES_CLUSTER_CONFIG_FILE_NAME = 'cfy_three_nodes_cluster_config.yaml'
+CLUSTER_CONFIG_FILE_NAME = 'cfy_cluster_config.yaml'
 CLUSTER_INSTALL_CONFIG_PATH = join(os.getcwd(), CLUSTER_CONFIG_FILE_NAME)
```

Comment:
It looks like you maybe don't use the _PATH any more, as you do this joining later?

Reply:
I'm not sure I understand where is the other joining.

Reply:
It and the other one are generated with a couple of joins here: https://github.com/cloudify-cosmo/cloudify-cluster-setup/pull/5/files/eb8099aaa6c5a3117f6e74f1dbaad17e2ade0822#diff-6adf86a90b46c627f7c336703696907bR421

Therefore, there's probably not much point having CLUSTER_INSTALL_CONFIG_PATH (unless you're planning to have THREE_NODES_CLUSTER_INSTALL_CONFIG_PATH too).

Reply:
Oh, I see. Well the CLUSTER_INSTALL_CONFIG_PATH specifies the output path of the generated configuration file. The joining afterwards is done in order to get the right configuration file template from the configuration files' templates directory. This template will be copied to the CLUSTER_INSTALL_CONFIG_PATH unless specified differently in the command (`--output`).

Reply:
Ah, fair enough- I must've missed that. Will green then.